tools/ci_build/github/azure-pipelines/post-merge-jobs.yml

```diff
@@ -24,22 +24,26 @@ jobs:
           #Merge the mulitple prof data into a single indexed profile data file
           llvm-profdata merge -sparse -o ort.profdata *.profraw
           #Create coverage report, output the result to 'report.json'
-          llvm-cov export -summary-only -instr-profile=ort.profdata onnxruntime_test_all -object onnxruntime_mlas_test  $(Build.SourcesDirectory)/include/onnxruntime $(Build.SourcesDirectory)/onnxruntime/core $(Build.SourcesDirectory)/onnxruntime/contrib_ops > $(Build.BinariesDirectory)/report.json
```

Comment:
>          llvm-cov export -summary-only -instr-profile=ort.profdata onnxruntime_test_all -object onnxruntime_mlas_test   [](http://example.com/codeflow?start=0&length=120)

Are the onnxruntime_mlas_test tests included indirectly now?

Reply:
Do you mean I should remove the following line: 
https://github.com/microsoft/onnxruntime/blob/df57e866228f096ea975d92f27eb0cffbe058d0b/tools/ci_build/github/azure-pipelines/post-merge-jobs.yml#L23

Reply:
I was just checking that onnxruntime_mlas_test was still run and included in the results. Wasn't clear to me why we replaced the calls to llvm-cov with onnxruntime_mlas_test with onnxruntime_api_tests_without_env.

Reply:
Oh, that's a mistake!